implementations/rust/common/Cargo.toml

```diff
@@ -1,9 +1,7 @@
 [package]
-name = "common"
+name = "ockam-common"
```

Comment:
My experience has been that modules called "common", "shared" etc. by definition tend to violate the single responsibility principle. They change a lot, for lot of reasons. This usually worsens the modularity of the code and increases coupling.

I think, it would be better to have an error module even if it is rather simple.

Reply:
Okay.